src/engine/prague.md

```diff
@@ -35,7 +35,7 @@ Method parameter list is extended with `executionRequests`.
   1. `executionPayload`: [`ExecutionPayloadV3`](./cancun.md#executionpayloadv3).
   2. `expectedBlobVersionedHashes`: `Array of DATA`, 32 Bytes - Array of expected blob versioned hashes to validate.
   3. `parentBeaconBlockRoot`: `DATA`, 32 Bytes - Root of the parent beacon block.
-  4. `executionRequests`: `Array of DATA` - List of execution layer triggered requests. Each list element is the corresponding request type's `request_data` as defined by [EIP-7685](https://eips.ethereum.org/EIPS/eip-7685). Elements of the list **MUST** be ordered by `request_type` in ascending order.
+  4. `executionRequests`: `Array of DATA` - List of execution layer triggered requests. Each list element is a `requests` byte array as defined by [EIP-7685](https://eips.ethereum.org/EIPS/eip-7685). The first byte of each element is the `request_type` and the remaining bytes are the `request_data`. Elements of the list **MUST** be ordered by `request_type` in ascending order. Elements with empty `request_data` are excluded from the list.
```

Comment:
> Elements with empty `request_data` are excluded from the list.

Is it worth mentioning in the case that all 3 (deposit, withdrawal, consolidation) requests are missing, `executionRequests` will still contain an empty array?

Reply:
Do you feel strongly that it has to be mentioned? All other parameters are also mandatory, but we do not write "`parentBeaconBlockRoot` must not be `null`" either. I feel that the type, `Array of DATA`, already means it will always be present and a JSON array.